code/modules/surgery/other.dm

```diff
@@ -141,6 +141,10 @@
 	user.visible_message("<span class='notice'> [user] has cut away necrotic tissue in [target]'s [affected.name] with \the [tool].</span>", \
 		"<span class='notice'> You have cut away necrotic tissue in [target]'s [affected.name] with \the [tool].</span>")
 	affected.open = 3
+	if(affected.status &= ORGAN_BROKEN)
```

Comment:
What are you trying to check here?

Reply:
It autocorrected to the wrong one.... This was meant to be burnt